users/create_user.go

```diff
@@ -98,7 +98,7 @@ func getTemporaryPassword(emailMessage string) (string, error) {
 	return subMatches[0][1], nil
 }
 
-var temporaryPasswordRegexp = regexp.MustCompile(`password=(\S+)\" style`)
+var temporaryPasswordRegexp = regexp.MustCompile(`password=(\S+)\"\s+\S+`)
```

Comment:
Something like this should be more robust and future proof,
```suggestion
// Matching links like https://digital-services.skf.com/sign-in?user_name=<username>&password=<temporary_password>
var temporaryPasswordRegexp = regexp.MustCompile(`\?(?:[^&]+&)*password=([^\"&]+)(?:\"|&)`)
```

Reply:
Some unit tests might be nice to add

Reply:
I changed regex as you suggested with aslight change to error if password is starting with space. I added some unitest.

Reply:
No need for password rule checks. reverted change on the regex.